dt-assets/functions/enqueue-scripts.php

```diff
@@ -197,9 +197,15 @@ function dt_site_scripts() {
                         "delete" => strtolower( __( "Delete", "disciple_tools" ) ),
                         "translate" => __( "Translate with Google Translate", "disciple_tools" ),
                         "hide_translation" => __( "Hide Translation", "disciple_tools" ),
+                        "reaction_title_1" => _x( '{{user}} reacted with {{emojiName}} emoji', 'Bob reacted with heart emoji (dont translate words in brackets)', 'disciple_tools' ),
+                        "reaction_title_many" => _x( '{{users}} and {{user}} reacted with {{emojiName}} emoji', 'Bob, Bill and Ben reacted with heart emoji (dont translate words in brackets)', 'disciple_tools' ),
                     ],
                     'current_user_id' => get_current_user_id(),
                     'additional_sections' => apply_filters( 'dt_comments_additional_sections', [], $post_type ),
+                    /**
+                     * reaction aliases must be lowercase with no spaces
```

Comment:
Just out of curiosity why do the aliases need to be lowercased?

Reply:
The dev @corsacca decided that all keys in D.T should be snake_case for consistency.